components/core/src/clp/streaming_archive/writer/Archive.hpp

```diff
@@ -279,6 +279,11 @@ class Archive {
      */
     void update_metadata();
 
+    /**
+     * Updates metadata in the global metadata database.
```

Comment:
```suggestion
     * Updates the archive's metadata in the global metadata database.
```

And we should probably update `update_metadata`'s docstring to something similar but replacing "global" with "local".